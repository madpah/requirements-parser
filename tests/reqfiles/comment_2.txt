req==1.0    # comment
