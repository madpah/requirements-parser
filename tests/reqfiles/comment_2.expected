[
 {
  "specifier": true, 
  "local_file": false, 
  "name": "req", 
  "editable": false, 
  "uri": null, 
  "extras": [], 
  "vcs": null, 
  "path": null, 
  "line": "req==1.0    # comment", 
  "specs": [
   [
    "==", 
    "1.0"
   ]
  ], 
  "revision": null
 }
]